January/26/dave_gelperin/info.yaml

```diff
@@ -0,0 +1,13 @@
+title: David Gelperin
+description: |
+    On January 26, 1940 was born David Gelperin.
+    He is an software engineer and recieved PhD in Computer Science from Ohio State University.
+    Gelperin and Hetzel had developed the STEP methodology for implementing the original IEEE-829 Standard for Test Documentation. 
+    In 1992 Gelperin and Hetzel firm organized the first "Software Testing, Analysis & Review", the "STAR" conference, in Las Vegas, Nevada
+    and in 1993 introduced a European equivalent, the EuroSTAR Conference.
+    He has written articles on software engineering, testing, and requirements, available at www.researchgate.net/profile/David_Gelperin/contributions.
+url: https://en.wikipedia.org/wiki/Dave_Gelperin
+images:
+    - filename: dave_gelperin.png
+      license: proprietary
+      source_url: https://images-na.ssl-images-amazon.com/images/I/A1BXdmEWlEL._SX450_.jpg
```

Comment:
We need the URL to the page, not the file itself. That's `https://www.amazon.com/David-Gelperin/e/B0765MCWPM`, see https://github.com/kiwitcms/history-of-testing/issues/43#issuecomment-1017189178

Reply:
We have link to the file itself for a lot of others, do not understand why now is different ?

Reply:
> We have link to the file itself for a lot of others

This statement is false. You can inspect the other yaml files and see for yourself that `source_url` always points to a page containing the image.

```
August/17/roy_osherove/info.yaml:      source_url: https://www.youtube.com/watch?v=sSdMcqxkEdo
Debug/33/loremipsum/info.yaml:      source_url: https://en.wikipedia.org/wiki/File:Lorem_ipsum_design.svg
December/10/ada_lovelace/info.yaml:      source_url: https://en.wikipedia.org/wiki/File:Ada_Lovelace_portrait.jpg
December/09/grace_hopper/info.yaml:      source_url: https://en.wikipedia.org/wiki/Grace_Hopper#/media/File:Commodore_Grace_M._Hopper,_USN_(covered).jpg
December/05/uncle_bob_martin/info.yaml:      source_url: https://en.wikipedia.org/wiki/File:Robert_C._Martin_surrounded_by_computers.jpg
December/12/glenford_myers/info.yaml:      source_url: https://www.yegor256.com/2014/08/22/art-of-software-testing.html
July/08/cem_kaner/info.yaml:      source_url: https://www.logigear.com/magazine/test-methods-and-metrics/dr-cem-kaner-shares-his-experiences-on-software-testing-and-essential-skills-needed-to-be-software-tester/
July/16/rex_black/info.yaml:      source_url: https://www.istqbexperttraining.com/images/rex_black_profil.jpg
June/11/dorothy_graham/info.yaml:      source_url: https://www.stickyminds.com/users/dorothy-graham
June/25/boris_beizer/info.yaml:      source_url: https://www.facebook.com/oz.code.software/photos/basw.AboSQKyxv0jUIWcPytVa-VyKuHVKIP_hCW3Yz0d8yN7QPJBfYkS1Vz6SJKxsTp1Kt0-Kgqcvq0ZM71dxeo1FYmB3HZEjmq2Q2xslyuDkZhqJUaMGawl4_1BY8zh04U1nXZJiJB5lL4wJ36gAQsFt_ArA/1468426496670465/
March/31/kent_beck/info.yaml:      source_url: https://commons.wikimedia.org/wiki/File:Kent_Beck_no_Workshop_Mapping_XP.jpg
March/13/erich_gamma/info.yaml:      source_url: https://devxconf.org/speakers/erich-gamma.jpg
May/26/ward_cunningham/info.yaml:      source_url: https://en.wikipedia.org/wiki/File:Ward_Cunningham_-_Commons-1.jpg
October/09/michael_wansley/info.yaml:      source_url: https://commons.wikimedia.org/wiki/File:Macklemore_(8465180525).jpg
September/09/international_software_testing_day/info.yaml:      source_url: https://commons.wikimedia.org/wiki/File:First_Computer_Bug,_1945.jpg
September/09/michael_hennell/info.yaml:      source_url: https://www.electronicsforu.com/technology-trends/embedded-safety-security-summit-esss
```

For example `https://en.wikipedia.org/wiki/File:Ward_Cunningham_-_Commons-1.jpg` is a page, while `https://upload.wikimedia.org/wikipedia/commons/thumb/8/88/Ward_Cunningham_-_Commons-1.jpg/717px-Ward_Cunningham_-_Commons-1.jpg` is the URL to a file (one resolution selected from several available).

Reply:
As I can see, March/13/erich_gamma/info.yaml:      source_url: https://devxconf.org/speakers/erich-gamma.jpg is link to the image, 
as well for example July/16/rex_black/info.yaml:      source_url: https://www.istqbexperttraining.com/images/rex_black_profil.jpg

Reply:
These are mistakes. Fixed in a373281fb4ed7cb12f50f8b33b418204fb00a381